tests/test_jump.py

```diff
@@ -485,42 +484,6 @@ def test_find_faint_extended_sigclip():
     #  Check that the flags are not applied in the 3rd group after the event
     assert np.all(gdq[0, 4, 12:22, 14:23]) == 0
 
-# No shower is found because the event is identical in all ints
-def test_find_faint_extended_sigclip():
```

Comment:
This test is duplicated. See:
https://github.com/spacetelescope/stcal/blob/787aa81a6719bb0c72be8972171c7f3f88c8adf0/tests/test_jump.py#L439
and
https://github.com/spacetelescope/stcal/blob/787aa81a6719bb0c72be8972171c7f3f88c8adf0/tests/test_jump.py#L489